src/index.ts

```diff
@@ -23,10 +65,23 @@ swdcTracker.initialize = async (swdcApiHost: string, namespace: string, appId: s
     const tracker_url_scheme = result.data.tracker_url_scheme || "https";
     // initialize snowplow tracker
     // endpoint, protocol, optional port, method, buffer events, request callback
-    const e = emitter(tracker_api_host, tracker_url_scheme, null, "post", 0, function (error: any, body: any, response: any) {
-      if (error) {
-        console.log("swdc-tracker collector stream error", error);
+    const e = emitter(tracker_api_host, tracker_url_scheme, null, "post", 0, function (err: any, body: any, response: any) {
+      let resp: any;
+      if (err) {
+        const errMsg = `swdc-tracker event error. ${err.message}`;
+        console.log(errMsg);
```

Comment:
want to use `console.error` here?

Reply:
yeah, it should use error. i'll update it

Reply:
updated